core-bundle/composer.json

```diff
@@ -141,6 +140,7 @@
         "symfony/security-csrf": "^6.4",
         "symfony/security-http": "^6.4",
         "symfony/service-contracts": "^3.0",
+        "symfony/stopwatch": "^6.4",
```

Comment:
This should not be a hard dependency of the `contao/core-bundle`. It's an optional dependency of the `contao/core-bundle` and we make it a hard dependency for the `contao/managed-edition` via the `contao/manager-bundle` instead.

Reply:
Fixed.